tests/unit/components/HdIcon.spec.js

```diff
@@ -111,6 +111,9 @@ describe('HdIcon', () => {
 
     await wrapper.vm.$nextTick();
 
-    expect(wrapper.html()).toMatchSnapshot();
+    const titleTag = wrapper.find('title')
+    const descTag = wrapper.find('desc')
+    expect(titleTag.element.id).toMatch(/\d+-title/)
+    expect(descTag.element.id).toMatch(/\d+-desc/)
```

Comment:
Ohhh, the random is biting here, right? 🙈

Reply:
Yeah, it didn't fail in the first run :D